lua/nvim-treesitter/parsers.lua

```diff
@@ -959,6 +959,15 @@ list.lalrpop = {
   maintainers = { "@traxys" },
 }
 
+list.solidity = {
+  install_info = {
+    url = "https://github.com/YongJieYongJie/tree-sitter-solidity",
+    branch = "with-generated-c-code",
```

Comment:
are you planning to maintain this branch?

Reply:
@theHamsta yes, I'll be maintaining this branch, keeping the generated C code updated with master (and upstream).